SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
Sure that will work. You'll need more documentation as that was not intuitive at all.